### PR TITLE
RHDEVDOCS-4497:-  Release note for doc: Improve awareness of resource/project limits and quota issues for developers

### PR DESCRIPTION
**Pull request by smunje1**
[RHDEVDOCS-4497](https://issues.redhat.com//browse/RHDEVDOCS-4497): Create release note for Doc: Improve awareness of resource/project limits and quota issues for developers

Aligned team: Dev Tools
OCP version for cherry-picking: enterprise-4.12 only
JIRA issues: [RHDEVDOCS-4497](https://issues.redhat.com//browse/RHDEVDOCS-4497)
Preview pages: [Download to see the preview in your browser](https://drive.google.com/file/d/1V08NgnRXLt-9Yiz6QorMNAmr_jA103KY/view?usp=share_link)
SME review needed: @divyanshiGupta 
QE review needed: @sanketpathak
Peer review needed: @deerskindoll

### CODE REVIEW

**Comment by smunje1** [comment]
Hello @divyanshiGupta 
Could you help with the SME review for this PR?

Thank you!

**Comment by smunje1** [comment]
Screenshot:
image

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -65,6 +65,7 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 * With this update, you can export your application in the ZIP file format to another project or cluster using the *Export application* option on the *+Add* page. 
 * With this update, you can create a Kafka event sink to receive events from a particular source and send them to a Kafka topic.
 * With this update, you can set the default resource preference in the *User Preferences* -> *Applications* page. In addition, you can select another resource type to be the default.
+* With this update, you can see the labels regarding quota status in the *Topology* view, which includes an alert to exceeding quotas.
```
Comment (divyanshiGupta) [inline release_notes/ocp-4-12-release-notes.adoc:69]: @smunje there are two parts to this epic:
Resource quota alert label on the Topology and Add page:
1. The above alert takes the user to the search page with all rqs and if there is just one rq then it takes the user to the details page for the same.

For deployments, if there are any errors associated with resource quotas, a warning alert is shown in the side panel.
Also a yellow border is shown around the deployment nodes when resource quota exceeds.

**Comment by smunje1** [comment]
Hello @divyanshiGupta 
Thank you for your review. I have incorporated your feedback. Could you please take a look?

Thank you!

**Comment by smunje1** [comment]
Screenshot:
image

File: release_notes/ocp-4-12-release-notes.adoc [hunk 2]
```diff
@@ -72,6 +72,8 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 * With this update, you can create a Kafka event sink to receive events from a particular source and send them to a Kafka topic.
 * With this update, you can set the default resource preference in the *User Preferences* -> *Applications* page. In addition, you can select another resource type to be the default.
 * With this update, you can make the `status.HostIP` node IP address for pods visible in the *Details* tab of the *Pods* page.
+* With this update, you can see the resource quota alert label on the *Topology* and *Add* pages whenever any resource reaches the quota. The alert label link takes the user to the *ResourceQuotas* search page having all resource quotas lists. If the alert label link is for a single resource quota, it takes the user to the *ResourceQuota details* page.
```
Comment (divyanshiGupta) [inline release_notes/ocp-4-12-release-notes.adoc:75]:
```suggestion
* With this update, you can see the resource quota alert label on the *Topology* and *Add* pages whenever any resource reaches the quota. The alert label link takes the user to the *ResourceQuotas* list page. If the alert label link is for a single resource quota, it takes the user to the *ResourceQuota details* page. 
```

File: release_notes/ocp-4-12-release-notes.adoc [hunk 3]
```diff
@@ -72,6 +72,8 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 * With this update, you can create a Kafka event sink to receive events from a particular source and send them to a Kafka topic.
 * With this update, you can set the default resource preference in the *User Preferences* -> *Applications* page. In addition, you can select another resource type to be the default.
 * With this update, you can make the `status.HostIP` node IP address for pods visible in the *Details* tab of the *Pods* page.
+* With this update, you can see the resource quota alert label on the *Topology* and *Add* pages whenever any resource reaches the quota. The alert label link takes the user to the *ResourceQuotas* search page having all resource quotas lists. If the alert label link is for a single resource quota, it takes the user to the *ResourceQuota details* page. 
+** For deployments, an alert is displayed in the topology node side panel if any errors are associated with resource quotas. Also, when the resource quota exceeds, a yellow border is displayed for the deployment nodes.
```
Comment (divyanshiGupta) [inline release_notes/ocp-4-12-release-notes.adoc:76]:
```suggestion
** For deployments, an alert is displayed in the topology node side panel if any errors are associated with resource quotas. Also, when the resource quota exceeds, a yellow border is displayed around the deployment nodes.
```

**Comment by smunje1** [comment]
@divyanshiGupta I have made changes as per your suggestion. Could you please confirm? Thank you!

**Comment by smunje1** [comment]
Screenshot:
image

**Comment by divyanshiGupta** [comment]
LGTM.

**Comment by smunje1** [comment]
Hello @sanketpathak 
Could you help with the QE review for this PR ?

Thank you!

**Comment by smunje1** [comment]
After rebasing and resolving merge conflicts, a screenshot of the content updates by following the new format for Release Notes as per the PR: https://github.com/openshift/openshift-docs/pull/51966,
image

Cc @divyanshiGupta

File: release_notes/ocp-4-12-release-notes.adoc [hunk 4]
```diff
@@ -96,6 +96,9 @@ With this release, there are several updates to the *Developer* perspective of t
 * Create a Kafka event sink to receive events from a particular source and send them to a Kafka topic.
 * Set the default resource preference in the *User Preferences* -> *Applications* page. In addition, you can select another resource type to be the default.
 * Make the `status.HostIP` node IP address for pods visible in the *Details* tab of the *Pods* page.
+* See the resource quota alert label on the *Topology* and *Add* pages whenever any resource reaches the quota. The alert label link takes the user to the *ResourceQuotas* list page. If the alert label link is for a single resource quota, it takes the user to the *ResourceQuota details* page. 
+** For deployments, an alert is displayed in the topology node side panel if any errors are associated with resource quotas. Also, when the resource quota exceeds, a yellow border is displayed for the deployment nodes.
```
Comment (divyanshiGupta) [inline release_notes/ocp-4-12-release-notes.adoc:100]:
```suggestion
** For deployments, an alert is displayed in the topology node side panel if any errors are associated with resource quotas. Also, when the resource quota exceeds, a yellow border is displayed around the deployment nodes.
```

**Comment by divyanshiGupta** [comment]
LGTM.

**Comment by smunje1** [comment]
> LGTM.

@divyanshiGupta Only Removed extra space on Line 101 as part of this push.

**Comment by sanketpathak** [comment]
LGTM

File: release_notes/ocp-4-12-release-notes.adoc [hunk 5]
```diff
@@ -96,6 +96,8 @@ With this release, there are several updates to the *Developer* perspective of t
 * Create a Kafka event sink to receive events from a particular source and send them to a Kafka topic.
 * Set the default resource preference in the *User Preferences* -> *Applications* page. In addition, you can select another resource type to be the default.
 * Make the `status.HostIP` node IP address for pods visible in the *Details* tab of the *Pods* page.
+* See the resource quota alert label on the *Topology* and *Add* pages whenever any resource reaches the quota. The alert label link takes the user to the *ResourceQuotas* list page. If the alert label link is for a single resource quota, it takes the user to the *ResourceQuota details* page.
```
Comment (deerskindoll) [inline release_notes/ocp-4-12-release-notes.adoc:99]:
```suggestion
* See the resource quota alert label on the *Topology* and *Add* pages whenever any resource reaches the quota. The alert label link takes you to the *ResourceQuotas* list page. If the alert label link is for a single resource quota, it takes you to the *ResourceQuota details* page. 
```

**Review by deerskindoll — COMMENTED** [review]
a couple minor suggestions added. let me know what you think.

**Comment by smunje1** [comment]
Hello @deerskindoll

Thanks for your suggestion. I have incorporated your feedback. Could you please confirm and send your approvals?

**Comment by smunje1** [comment]
After updates, a screenshot:
image

**Review by deerskindoll — APPROVED** [review]
lgtm

**Pull request merged**